invenio_vocabularies/config.py

```diff
@@ -104,14 +105,19 @@
 }
 """Names allowed identifier schemes."""
 
-# configure CUSTOM_VOCABULARY_TYPES to differentiate output. Is used in VocabulariesServiceConfig
+VOCABULARIES_SUBJECTS_SCHEMES = {
+    "gnd": {"label": _("GND"), "validator": idutils.is_gnd, "datacite": "GND"},
```

Comment:
minor: we should also have a `mesh` scheme, though it has to be implemented in `idutils`. It would just be `mesh:D000001`, but it would allow to store a more compact version of the identifier instead of the full URL, and generate the fully resolvable URL through it